ThePorch/src/AppStateTracker.js

```diff
@@ -0,0 +1,51 @@
+import { Component } from 'react';
+import { AppState } from 'react-native';
+import PropTypes from 'prop-types';
+
+class AppStateTracker extends Component {
+  state = {
+    appState: AppState.currentState,
+    currentTime: new Date().getTime(),
```

Comment:
Doesn't amplitude track time automatically? I didn't think we needed to log this on our own

Reply:
From their docs...

> Client-side: By default, using our JavaScript SDK, events triggered within 30 minutes of each other are counted towards the current session (the same Session ID). The time of the first event marks the start time of a session and the last event triggered marks the end time of a session

They do, this will be a tiny bit more accurate. I'm happy to gut this if you think it's unneeded.